components/Staking/StakingStepper.jsx

```diff
@@ -48,6 +49,20 @@ const SetUpAndStake = ({ disabled, twitterId, multisigAddress, onNextStep }) =>
   const { chainId, address: account } = useAccount();
   const { switchChainAsync, switchChain } = useSwitchChain();
 
+  const { data: serviceInfo, isLoading: isServiceInfoLoading } = useAccountServiceInfo({ account })
```

Comment:
this is for getting staking contract name if a user staked and got back to this page